docs/quick_start.rst

```diff
@@ -64,10 +65,137 @@ In general, Fledge installation will require the following packages:
 - One or more Fledge South services
 - One or more Fledge North service (OSI PI and OCS north services are included in Fledge core)
 
-Installing Fledge packages
-###########################
+Using the package repository to install Fledge
+##############################################
 
-SSH into the system that will host Fledge services. For each Fledge package that you choose to install, type the following command::
+If you choose to use the Dianomic Systems package repository to install the packages you will need to follow the steps outlined below for the particular platform you are using.
+
+Ubuntu or Debian
+~~~~~~~~~~~~~~~~
+
+On a Ubuntu or Debian system, including the Raspberry Pi, the package manager that is supported in *apt*. You will need to add the Dianomic Systems archive server into the configuration of apt on your system. The first thing that most be done is to add the key that is used to verify the package repository. To do this run the command
+
+.. code-block:: console
+
+   wget -q -O - http://archives.fledge-iot.org/KEY.gpg | sudo apt-key add -
+
+Once complete you can add the repository itself into the apt configuration file /etc/apt/sources.list. The simplest way to do this is the use the *add-apt-repository* command. The exact command will vary between systems;
+
+  - Raspberry Pi does not have an apt-add-repository command, the user must edit the apt sources file manually
+
+    .. code-block:: console
+
+       sudo vi /etc/apt/sources.list
+       
+    and add the line
+    
+    .. code-block:: console
+
+       deb  http://archives.fledge-iot.org/latest/buster/arm7l/ /
```

Comment:
Also I see foglamp packages which is available under at `foglamp` directory NOT at the root level. Don't you think it should directly inside the `archives.dianomic.com/`? Otherwise special case needs to be added at white-labelled documentation side.

Reply:
Yes it should. We want to avoid the special cases.

Reply:
cc/ @YashTatkondawar @praveen-garg

Reply:
https://github.com/dianomic/FogLAMP-release/pull/22/files will resolve this too. We should keep the foglamp directory. It was kept separate to allow other product packages too especially **/fogman** binaries

Reply:
if that's the case then change required in `foglamp-documentation` repo

Reply:
Added https://github.com/dianomic/foglamp-documentation/pull/5 
I was assuming the pipeline has FogLAMP-release and then pass that to foglamp-documention. Probably we should do that way to avoid scripting duplication and put the processed source code into readthedocs server and archives repo from that final output only.  (instead of baking partially on both)

Reply:
> https://github.com/dianomic/FogLAMP-release/pull/22/files will resolve this too. We should keep the foglamp directory. It was kept separate to allow other product packages too especially **/fogman** binaries

Currently we are not planning to do this, but you are right we should leave scope for it. @Ashish, let's do that as part of the processing of the FogLAMP variant.